src/hdmf/term_set.py

```diff
@@ -55,9 +58,14 @@ def __init__(self,
 
         self.sources = self.view.schema.prefixes
 
-    def __repr__(self):
-        re = "class: %s\n" % str(self.__class__)
-        re += "term_schema_path: %s\n" % self.term_schema_path
+    def __repr__(self): # pragma: no cover
+        re = "Class: %s\n" % str(self.__class__.__name__)
+        re += "Schema Path: %s\n" % self.term_schema_path
+        return re
+
+    def _repr_html_(self): # pragma: no cover
+        re = "<p><b>" + "Schema Path: " + "</b>" + self.term_schema_path + "<br>" + "</p>"
+        re += "<p><b>" + "Sources: " + "</b>" + ", ".join(list(self.sources.keys())) + "</p>"
```

Comment:
1. Could you add a screenshot in the PR showing how this looks?
2. This and `__repr__` are testable. Please do not exclude lines of code from coverage checks unless there is a very good reason why that code will not ever run, e.g., some code exists as an additional just-in-case error check but if everything is configured by HDMF correctly, that error should never run. In this case, the code is very likely to run and should be tested to make sure there is not a silly error within it.
3. I think `__repr__` and `_repr_html_` should display the same information. Showing the sources is nice - why not add that to `__repr__`?

Reply:
1. I can add a screenshot.
2. We didn't have tests before for them but we should? It seems a bit unnecessary for these methods.
3. I can make them the same, but I treated them differently based on where I interact with the class. In terminal it was more useful for me to see the path than the sources themselves. I think what I'll do is make sure both representations have sources and path.

Reply:
![Uploading Screenshot 2023-10-19 at 7.32.44 AM.png…]()

Reply:
re: 2. I think we should. If the code were as simple as just returning the name, then maybe not. Here, you have slightly more complex code where you could have had a typo, used += instead of =, the `join` used the wrong syntax, self.sources might not have been initialized, etc. Better to spend the 3 minutes to write at least one test to make sure the code runs as expected.

Reply:
re: 2. I think we should. If the code were as simple as just returning the name, then maybe not. Here, you have slightly more complex code where you could have had a typo, used `=` instead of `+=`, the `join` used the wrong syntax, `self.sources` might not have been initialized, etc. Better to spend the 3 minutes to write at least one test to make sure the code runs as expected than to have a user discover the bug down the road.